modules/ROOT/pages/notifications/gqlstatus-info-codes.adoc

```diff
@@ -0,0 +1,62 @@
+:description: The page contains GQLSTATUS info codes producing results such as SUCCESS, NO DATA, and OMITTED RESULT.
+
+[[gql-status-info-codes]]
+= Primary GQLSTATUS codes
+
+GQL requires several new statuses that are not covered by the current Neo4j notification framework.
+
+.GQL status information codes
+[cols="1,1,1,1", options="header"]
+|===
+|GQLSTATUS
+|Condition
+|Subcondition
+|Description
+
+| 00000
+| Successful completion
+|
+| Successful completion with a regular non-empty result
+
+| 00001
+| Successful completion
+| Omitted result
+| Successful completion with no return columns (e.g., update with no `RETURN` clause)
+
+| 02000
+| No data
+| 
+| Successful completion with an empty result 
```

Comment:
Potentially we could clarify that this means 'no result rows' (in contrast to 'no result columns' for 00001). But on the other hand, 'empty result' is an established concept which we use elsewhere in docs. I leave it up to you to decide